cmd/prometheus/agent_mode.go

```diff
@@ -0,0 +1,7 @@
+//+build agent
+
+package main
+
+func isAgent() bool {
```

Comment:
I initially did this as an init var, but then I switched to a function instead to make the initialization order less confusing.